queue/config.go

```diff
@@ -13,14 +13,14 @@ import (
 )
 
 type Config struct {
-	MemSettings  *memqueue.Settings  `config:"memqueue"`
-	DiskSettings *diskqueue.Settings `config:"diskqueue"`
+	MemSettings  *memqueue.Settings  `config:"mem"`
+	DiskSettings *diskqueue.Settings `config:"disk"`
 }
 
 func DefaultConfig() Config {
 	return Config{
 		MemSettings: &memqueue.Settings{
-			Events:         1024,
+			Events:         4096,
```

Comment:
Is this choice based on something?

Reply:
It is the the current default for beats run under agent. I will add a comment since it seems arbitrary.